package.json

```diff
@@ -99,7 +99,7 @@
 		"generate": "generate-calypso-package",
 		"install-if-no-packages": "node bin/install-if-no-packages.js",
 		"lint": "run-s -s 'lint:*'",
-		"lint:config-defaults": "node bin/validate-config-keys.js",
+		"lint:config-defaults": "node --experimental-strip-types bin/validate-config-keys.mts",
```

Comment:
Note that while this is "experimental" here, [the feature is already stable in the next version of Node.js ](https://nodejs.org/en/blog/release/v23.6.0).